core/plugins/org.polarsys.capella.core.data.fa.ui.quickfix/src/org/polarsys/capella/core/data/fa/ui/quickfix/resolver/DCOM_23_Resolver_Source.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+
+package org.polarsys.capella.core.data.fa.ui.quickfix.resolver;
+
+import java.util.Collection;
+
+import org.eclipse.core.resources.IMarker;
+import org.eclipse.emf.ecore.EObject;
+import org.polarsys.capella.core.data.fa.FunctionOutputPort;
+import org.polarsys.capella.core.data.fa.FunctionalExchange;
+import org.polarsys.capella.core.data.fa.ui.quicfix.helpers.QuickFixNavigationHelper;
+import org.polarsys.capella.core.validation.ui.ide.quickfix.AbstractCapellaMarkerResolution;
+
+/**
+ * QuickFix allowing to show source Function Input Port in Project Explorer.
+ */
+public class DCOM_23_Resolver_Source extends AbstractCapellaMarkerResolution {
+
+  @Override
+  public void run(IMarker marker) {

```

Comment:
The two "show" resolver doesn"t apply on Operational analsys layer. There is a AbstractCapellaMarkerResolution.canResolve that might allow to make it not visible on Interactions